app/dcr_monitor/tee_backend/confidential_space.go

```diff
@@ -27,7 +29,15 @@ type TEEProviderGCPConfidentialSpace struct {
 	client    *compute.InstancesClient
 }
 
-func NewTEEProviderGCPConfidentialSpace(ctx context.Context, projectId string, region string, zone string, env string) (*TEEProviderGCPConfidentialSpace, error) {
+func NewTEEProviderGCPConfidentialSpace(ctx context.Context) (*TEEProviderGCPConfidentialSpace, error) {
+	env := os.Getenv("ENV")
+	projectId := os.Getenv("PROJECT_ID")
+	region := os.Getenv("REGION")
+	zone := os.Getenv("ZONE")
```

Comment:
Can you please check if the environment variables are set, and print error if they are not present?

Reply:
return err when the environment variables not set